packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -275,6 +287,16 @@ class _RenderCupertinoSwitch extends RenderConstrainedBox {
     markNeedsPaint();
   }
 
+  DragStartBehavior get dragStartBehavior => _dragStartBehavior;
+  DragStartBehavior _dragStartBehavior;
```

Comment:
it's not clear to me that we need this field. We can just have `dragStartBehavior` return `_drag.dragStartBehavior`, no?